cpp/memilio/math/integrator.h

```diff
@@ -112,8 +162,9 @@ class OdeIntegrator
         bool step_okay = true;
 
         FP dt_copy; // used to check whether step sizing is adaptive
-        FP dt_restore = 0; // used to restore dt if dt was decreased to reach tmax
-        FP t          = t0;
+        FP dt_restore  = 0.0; // used to restore dt, if dt was decreased to reach tmax
+        FP dt_min_copy = m_core->get_dt_min(); // used to restore dt_min, if it was decreased to reach tmax
```

Comment:
Since the comments are exactly the same, it is confusing that one is called restore and one is called copy.
If the comments remain the same, it should be called dt_min_restore